Mathlib/Combinatorics/SimpleGraph/Regularity/Bound.lean

```diff
@@ -267,24 +267,24 @@ theorem add_div_le_sum_sq_div_card (hst : s ⊆ t) (f : ι → 𝕜) (d : 𝕜)
 
 end SzemerediRegularity
 
-namespace Tactic
+namespace Mathlib.Meta.Positivity
```

Comment:
@YaelDillies, I assume this change is fine?

Reply:
Hmm, yeah. This is a misport, since the convention changed in Lean 3.